teps/0104-tasklevel-resource-requirements.md

```diff
@@ -192,8 +199,93 @@ If there is no container configuration that satisfies the LimitRange, the TaskRu
 - Tekton pods currently have a [burstable quality of service class][qos], which will not change as a result of this implementation.
 - We should consider updating our catalog Task guidelines with guidance around when to specify resource requirements. 
 
+### Future Work
+
+We should consider deprecating `Task.Step.Resources`, `Task.StepTemplate.Resources`, and `TaskRun.StepOverrides`.
+Specifying resource requirements for individual Steps is confusing and likely too granular for many CI/CD workflows.
+
+We could also consider support for both Task-level and Step-level resource requirements if the requirements are for different types
```

Comment:
@austinzhao-go updated the tep to clarify
```
We could also consider support for both Task-level and Step-level resource requirements if the requirements are for different types
of compute resources (for example, specifying CPU request at the Step level and memory request at the Task level). However,
this functionality will not be supported by the initial implementation of this proposal; it can be added later if desired.
```